src/core/lib/resource_quota/memory_quota.cc

```diff
@@ -371,7 +371,8 @@ void BasicMemoryQuota::Start() {
           std::tuple<const char*, RefCountedPtr<ReclaimerQueue::Handle>> arg) {
         auto reclaimer = std::move(std::get<1>(arg));
         if (GRPC_TRACE_FLAG_ENABLED(grpc_resource_quota_trace)) {
-          double free = std::max(intptr_t(0), self->free_bytes_.load());
+          double free =
+              std::max(static_cast<intptr_t>(0), self->free_bytes_.load());
```

Comment:
The changes in this file should use braces.

I'll stop noting these individually, but please fix throughout the PR.  Anywhere that the argument is a constant value, it's probably better to use braces than call it a cast.